src/components/ListTemplate/listTemplate.js

```diff
@@ -118,6 +118,22 @@ export const listTemplate = ({
 
     return <DateColumn value={updatedDate} />;
   },
+  status: record => {
+    const {
+      status,
+      progress,
+    } = record;
+
+    if (status === 'ERROR') {
```

Comment:
Consider moving this string to `constants`

Reply:
done